CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
-- Fix: POST /v2/registrations allows forbidden attribute names in NGSIv2 (#3468)
+
+- Fix: PATCH /v2/entities/ID/attrs error doesn't provide info about offending attributes (#2933)
```

Comment:
This seems to be a conflict not correctly solved...

The line `- POST /v2/registrations allows forbidden attribute names in NGSIv2 (#3468)` shouldn't be removed. Please recover it and just add the new one: `- Fix: PATCH /v2/entities/ID/attrs error doesn't provide info about offending attributes (#2933)`